corehq/apps/app_manager/views/releases.py

```diff
@@ -116,12 +120,13 @@ def paginate_releases(request, domain, app_id):
         for app in saved_apps:
             app['num_errors'] = num_errors_dict.get(app['version'], 0)
 
-    num_pages = int(ceil(apps.total / limit))
+    total_apps = app_es.count()
+    num_pages = int(ceil(total_apps / limit))
```

Comment:
Might be worth doing `float(total_apps)` here.  If `total_apps` and `limit` are both integers, this does integer division, and you'll end up with the rounded-down integer.  I'm not certain whether the old implementation returned a float for `apps.total` or if this was just broken.

Reply:
this file has `import division`, so this line does what it is supposed to.

Reply:
![future](https://78.media.tumblr.com/e1f164a4f3f6282b4a1300bbd4f29621/tumblr_nsdyp4wVJC1urp9z5o1_500.gif)

Reply:
oh wow, thats quite some mysterious stuff..